.github/workflows/release-please-mention-contributors.yml

```diff
@@ -0,0 +1,54 @@
+on:
+  pull_request:
+    paths:
+      - CHANGELOG.md
+
+name: "release-please: mention contributors"
+jobs:
+  mention-contributors:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/github-script@v6
+        id: find-authors
+        with:
+          script: |
+            const { stdout: diff } = await exec.getExecOutput("git diff origin/${{ github.base_ref }} -U0 CHANGELOG.md")
+            console.log({ diff })
+
+            const COMMIT_URL_REGEX = /\/commit\/(\w+)\)/g
+
+            const authors = []
+            for (const match of diff.matchAll(COMMIT_URL_REGEX)) {
+              const ref = match[1]
+              const [owner, repo] = "${{ github.repository }}".split("/")
+              const commit = await github.rest.repos.getCommit({ owner, repo, ref })
+              const { author } = commit.data
+              if (author) {
+                authors.push(author.login)
+              }
+            }
```

Comment:
maybe rewrite this `for` into a `Promise.all` with `map`?

Reply:
not all commits have an author (bots, for example, don't), so this would require `flatMap`, which would make things harder to understand. I think it's fine to use a `for` here.

Reply:
you'd just need `map(...).filter()` - not really hard to understand

Reply:
implemented in f22d5ceeea838a02ce7b44cce6a785eb4fbbc106